docker/amd/docker-compose.yml

```diff
@@ -4,42 +4,46 @@ services:
     build:
       context: .
       args:
-        # Requirements file to use: 
-        # | GPU | CPU | requirements file to use |
-        # |--------|---------|---------|
-        # | NVIDIA | has AVX2 | `requirements.txt` |
-        # | NVIDIA | no AVX2 | `requirements_noavx2.txt` |
-        # | AMD | has AVX2 | `requirements_amd.txt` |
-        # | AMD | no AVX2 | `requirements_amd_noavx2.txt` |
-        # | CPU only | has AVX2 | `requirements_cpu_only.txt` |
-        # | CPU only | no AVX2 | `requirements_cpu_only_noavx2.txt` |
-        # | Apple | Intel | `requirements_apple_intel.txt` |
-        # | Apple | Apple Silicon | `requirements_apple_silicon.txt` |
+        GPU_CHOICE: B # AMD
+
+        # Requirements file to use:
+        # | GPU      | CPU           | requirements file to use           |
+        # | -------- | ------------- | ---------------------------------- |
+        # | NVIDIA   | has AVX2      | `requirements.txt`                 |
+        # | NVIDIA   | no AVX2       | `requirements_noavx2.txt`          |
+        # | AMD      | has AVX2      | `requirements_amd.txt`             |
+        # | AMD      | no AVX2       | `requirements_amd_noavx2.txt`      |
+        # | CPU only | has AVX2      | `requirements_cpu_only.txt`        |
+        # | CPU only | no AVX2       | `requirements_cpu_only_noavx2.txt` |
+        # | Apple    | Intel         | `requirements_apple_intel.txt`     |
+        # | Apple    | Apple Silicon | `requirements_apple_silicon.txt`   |
         # Default: requirements.txt`
         # BUILD_REQUIREMENTS: requirements.txt
-        
-        # Extension requirements to build: 
-        # BUILD_EXTENSIONS: 
-        
-        # specify which cuda version your card supports: https://developer.nvidia.com/cuda-gpus
-        TORCH_CUDA_ARCH_LIST: ${TORCH_CUDA_ARCH_LIST:-7.5} 
+
+        # Extension requirements to build:
         BUILD_EXTENSIONS: ${BUILD_EXTENSIONS:-}
-        APP_GID: ${APP_GID:-6972} 
-        APP_UID: ${APP_UID-6972} 
+
+        # specify which cuda version your card supports: https://developer.nvidia.com/cuda-gpus
+        TORCH_CUDA_ARCH_LIST: ${TORCH_CUDA_ARCH_LIST:-7.5}
+        APP_GID: ${APP_GID:-6972}
+        APP_UID: ${APP_UID:-6972}
```

Comment:
Are these actually used for anything? Nothing appears to reference them: https://github.com/search?q=repo%3Aoobabooga%2Ftext-generation-webui%20APP_UID&type=code

If they're only meant to change the runtime user, this should just be done with the standard `user: uid:gid` compose setting or `docker run --user uid:gid` CLI arg.